RentallCarsAPI/Controllers/RentalController.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using RentallCarsAPI.Models;
+using RentallCarsAPI.Models.Request;
+using RentallCarsAPI.Models.Response;
+using RentallCarsAPI.Tools.Interfaces;
+
+namespace RentallCarsAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class RentalController : Controller
+    {
+        private readonly IRentalHelper _rentalHelper;
+        private readonly IConfiguration _configuration;
+
+        public RentalController(IRentalHelper iRentalHelper,IConfiguration iConfiguration)
+        {
+            _configuration = iConfiguration;
+            _rentalHelper = iRentalHelper;
+        }
+
+        [HttpPost]
+        public IActionResult Create(RentalRequest rentalRequest)
+        {
+            var rentals = _rentalHelper.GetAll();
+            var car = _rentalHelper.GetCar(rentalRequest.Car);
+            if (car == null)
+            {
+                return NotFound(new Response { Message = "Car not found"});
```

Comment:
Solo para tenerlo en cuenta a futuro. Si no me equivoco, tenemos esta posibilidad para estos casos:
```suggestion
                return NotFound("Car not found");
```

Reply:
okay perfecto... como ves eto para merge¿?